lib/wanda/checks_selection.ex

```diff
@@ -0,0 +1,78 @@
+defmodule Wanda.ChecksSelection do
+  @moduledoc """
+  Entry point to interact with a targeted checks selection.
+  """
+
+  alias Wanda.Catalog
+  alias Wanda.Catalog.{CheckCustomization, SelectableCheck, Value}
+  alias Wanda.ChecksCustomizations
+
+  @spec selectable_checks(group_id :: String.t(), env :: map()) :: [SelectableCheck.t()]
+  def selectable_checks(group_id, env) do
+    available_customizations = ChecksCustomizations.get_customizations(group_id)
+
+    env
+    |> Catalog.get_catalog()
+    |> Enum.map(
+      &%SelectableCheck{
+        id: &1.id,
+        name: &1.name,
+        group: &1.group,
+        description: &1.description,
+        values:
+          &1.id
+          |> find_custom_values(available_customizations)
+          |> map_values(&1.values),
+        customizable: &1.customizable
+      }
+    )
+  end
+
+  defp find_custom_values(check_id, available_customizations) do
+    Enum.find_value(available_customizations, [], fn
+      %CheckCustomization{check_id: ^check_id, custom_values: custom_values} ->
+        custom_values
+
+      _ ->
+        nil
```

Comment:
Should this not be a list  as map_values/2 expects  a list and in worst case it receives a nil

Reply:
`nil` here means _the customization was not found_.

When not found values falls back to an empty list as defined in the second argument of `Enum.find_value` on line 32
```elixir
Enum.find_value(available_customizations, [], fn ...
```